src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/chainedmethodcallwrap/InputChainedMethodCallWrapMaxSingleLineCalls.java

```diff
@@ -0,0 +1,27 @@
+/*
+ChainedMethodCallWrap
+identifierPattern = ^assert.*$
+maxCallsPerLine = 2
+maxSingleLineCalls = 2
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.chainedmethodcallwrap;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import com.google.common.truth.Truth;
+
+public class InputChainedMethodCallWrapMaxSingleLineCalls {
+
+    void method() {
+        assertWithMessage("test").that(1).isEqualTo(1);
+                           // violation above ''3' method calls on single line, max allowed is '2''
+        assertWithMessage("test")
+                .that(1).isNotEqualTo(2); // ok
+        Truth.assertWithMessage("test").that(1).isNotEqualTo(2);
+                                                       // ^---- ok, specified pattern is different
```

Comment:
Though the method being implied is the same, the check matches again the complete method chain, can be changed if asked for but then it will also match for-
```
SomeOtherBuilder.assertWithMessage().that.....
```
isn't that big of a problem IMO, the user can change this based on their use case.

Reply:
Please add this to documentation example